app/modules/visualization/controllers/visualizationControllers.js

```diff
@@ -791,75 +791,77 @@ angular.module('pcApp.visualization.controllers.visualization', [
-
-                        var from_country = '';
-                        var to_country = '';
-
-                        var initialZoomMap = 2;
-                        if (document.getElementById('initialZoom').value > 0) {
-                            initialZoomMap = document.getElementById('initialZoom').value;
-                        }
-
-                        if (document.getElementById('initialLat').value) {
-                            var initialLat = document.getElementById('initialLat').value;
-                        }
-
-                        if (document.getElementById('initialLng').value) {
-                            var initialLng = document.getElementById('initialLng').value;
-                        }
-
-                        if ($scope.list) {
-                            margin.top = margin.top / 5;
-                            margin.right = margin.right / 5;
-                            margin.bottom = margin.bottom / 5;
-                            margin.left = margin.left / 5;
-                            width = width / 5;
-                            height = height / 5;
-                            font_size = font_size / 5;
-                            $scope.showLegend = false;
-                            $scope.showZoom = false;
-                            $scope.showBubbles = false;
-                            $scope.showMovement = false;
-                            $scope.showBubbles = false;
-                            
-                        } else {
-                            from_country = $scope.translateCountryValue($scope.rangeDatesSliderMin);
-                            to_country = $scope.translateCountryValue($scope.rangeDatesSliderMin);
-                        }
-
-                        var mapObj = policycompass.viz.mapLeaflet({
-                            'idName': "container_graph_" + $scope.visualization.id,
-                            'width': width,
-                            'height': height,
-                            'margin': margin,
-                            'font_size': font_size,
-                            'mode': $scope.mode,
-                            'scaleColor': $scope.scaleColor,
-                            'legend': $scope.showLegend,
-                            'projection': $scope.typeToPlot,
-                            'showZoom': $scope.showZoom,
-                            'initialLat': initialLat,
-                            'initialLng': initialLng,
-                            'initialZoom': initialZoomMap,
-                            'showBubbles': $scope.showBubbles,
-                            'showMovement': $scope.showMovement,
-                            'data': $scope.datasetToSendMap,
-                            'from_country': from_country,
-                            'to_country': to_country
-                        });
-
+                    	if (!$routeParams.storyId>0) {

```

Comment:
please do not use $routeParams. Why do you need to check if the chart is (not) in a story? What will happen when used in an other URL where there isn't storyId parameter?
In general, you should not care where the chart is included. Consider adding options in the directive instead.